src/test/scala/com/microsoft/ml/spark/cognitive/split1/TextAnalyticsSDKSuite.scala

```diff
@@ -0,0 +1,56 @@
+package com.microsoft.ml.spark.cognitive.split1
+
+import com.azure.ai.textanalytics.models.TextAnalyticsRequestOptions
+import com.microsoft.ml.spark.cognitive.{CognitiveServicesBase, _}
+import com.microsoft.ml.spark.core.test.base.TestBase
+import com.microsoft.ml.spark.stages.FixedMiniBatchTransformer
+import org.apache.spark.ml.param.DataFrameEquality
+import org.apache.spark.sql.{DataFrame, Row}
+
+class TextAnalyticsSDKSuite extends TestBase with DataFrameEquality with TextKey {
+
+  import spark.implicits._
+
+  lazy val df: DataFrame = Seq(
+    "Hello World",
+    "Bonjour tout le monde",
+    "La carretera estaba atascada. Había mucho tráfico el día de ayer.",
+    ":) :( :D"
+  ).toDF("text2")
+
+  val options: Option[TextAnalyticsRequestOptions] = Some(new TextAnalyticsRequestOptions()
+    .setIncludeStatistics(true))
+
+  lazy val detector: TextAnalyticsLanguageDetection = new TextAnalyticsLanguageDetection(options)
+    .setSubscriptionKey(textKey)
+    .setEndpoint("https://ta-internshipconnector.cognitiveservices.azure.com/")

```

Comment:
I'm surprise you don't have a conflict here. In the latest commit in the remote branch this endpoint is already set to the correct region matching the `textKey`. No need to use the `ta-internshipconnector` TA resource.